nginx/servers/proximis.conf.dist

```diff
@@ -1,7 +1,7 @@
 server {
     listen 80;
 
-    server_name server_name.localhost;
+    server_name project-uuid.localhost;
```

Comment:
This is just a suggestion, but you can use `server_name ~^(?<domain>.+).localhost$;` to match your project uuid, then reuse the $domain var in your CHANGE_CONFIG_NAME and CHANGE_TMP